### DISCUSSION TITLE
gothic-win64-v1.0.1311

### DISCUSSION THREAD

**Discussion by Try** [Try/OpenGothic#207] [General]
![изображение](https://user-images.githubusercontent.com/122894/140609783-8ca548c2-794c-4b57-9726-0df93218f280.png)

* General
  * New save-file format (#202)
  * Improve camera behaviour, to match original game  (#198)
  * Added k-key functionality in Marvin Mode (#194)
  * Marvin mode is now disabled by default (use b-marvin-b to activate)
  * Added climb behaviour for AI
  * multiple CPU-performance improvals

* Game
  * Adjust melee combat to original game
  * Added theft reaction
  * Optimize sound occlusion calculation
  * Add dynamic multi line support for dialogs (#189)
  
* Script-api
  * Added wld_sendtrigger, zCTrigger.fireDelaySec
  * Added init_global

* Graphics
  * Added animation blending
  * Use DEFAULT.TGA as fallback for missing textures
  * Added wind animation
  * Added model-fatness
  
* MacOS
  * Fixed mouse-camera controls

<hr /><em>This discussion was created from the release <a href='https://github.com/Try/OpenGothic/releases/tag/gothic-win64-v1.0.1311'>gothic-win64-v1.0.1311</a>.</em>

**Comment by x4rd4s** [comment]
Edit: Sound is working after a reboot. Unfortunately, I cannot tell what went wrong in the first place. But I have noticed a different thing: while the original G2 intro video plays just fine, the addon video while talking to Xardas in the beginning is not working. Terminal tells me that it couldn't find the addonintro movie file, although it must be there since I could watch it when playing vanilla.

Thank you very much for all your effort! Very appreciated!

Camera behaviour is very alike to the original. I just noticed that the walking animation is a little bit clunky compared to original. I also have the problem that I don't have any sound (using Arch Linux). The terminal states nothing regarding my sound problem. I also needed to start Gothic 2 via Wine once in order for OpenGothic to start the game without error from the main menu.

**Reply by Try** [reply]
> Sound is working after a reboot.

:D

About video: there is actually 2 addon videos - one with Raven and another with NotR logo. In most releases the second video is simply absent - that is probably what console message was about.
I assume both videos were missing, right? Can you tell me what version(Steam/Gog/CD; RU/DE/EN/PL) of the game you playing?

**Reply by x4rd4s** [reply]
I am using the GOG version of the game in a wine-prefix.

Initially I could start OpenGothic but when I clicked start game in the main menu nothing happened. Only if I started Gothic 2 NotR vanilla once, OpenGothic was able to locate the intro movie files in order for it to properly run.

**Comment by x4rd4s** [comment]
Ok, I have tested OpenGothic for an hour or so and here is what I have noticed so far (I hope it is okay for you to post it here):

First of all my system specs:
OS: Arch Linux
Kernel: 5.14.16
CPU: Intel i5-6600k
GPU: NVidia GeForce 1070
GPU driver: nvidia 495.44
Memory: 16GB
Gothic 2 NotR installation: German GOG installation without any 3rd-party-patches in a wine prefix

- my first savegame got lost when I tried to play on the next day. But I couldn't reproduce this behaviour, so I am not sure what exactly went wrong
- weird bounce behaviour: tried to fall of a roof (Xardas tower, to get behind it for loot): got stuck on the edge of the roof and then got bounced off quite a bit which makes landing on a narrow ledge very cumbersome (couple of reloads until I got it)
- tried to jump next to a tree (to reach a ledge) but I hit the tree, got stuck in it and then got catapulted 70m into the air (quite funny, though :))
- when walking against small obstacle like branches and stuff, character bounces off sometimes, which makes him automatically turn around which is quite annoying 
- enemy AI seems to be a little off: combat feels too easy since AI is not fighting back properly
- (it seems that the enemy AI has some problems in narrow spaces when there is more then one enemy but I am not so sure about this)
- sometimes game is not focused properly when starting although I see the main menu. I have to alt-tab to it to make keyboard input work
- walking animation when using keyboard controls but more so when using mouse to look around is "clunky" and stuttery
- musik is okay but it seems like the mixing is not right (difference to vanilla songs is noticable)
- empty enemy corpses still get highlighted which makes looting enemies on top of each other painful
- items which fall and slide on the ground also slide on water surface
- no enemy infighting (e.g. wolf vs rat)
- Gothic.ini and save files are stored in the home directory of the system user instead of the game directory

I really do enjoy playing G2 with your engine and I am very impressed how far you have already come (been following this project for over a year now). My criticism is just meant to help you and others out and since I am not a well-enough programmer myself to fix bugs etc. I want to address as many problems as I can to help you out on this front. All your effort is very much appreciated!

**Reply by Try** [reply]
Generally can confirm most of cases, but still let me do a little breakdown:
> Gothic.ini and save files

Most likely it's something related to app work directory (what normally is not an issue on Windows) - I'll check.

> no enemy infighting

Tested today - infighting actually works. I've been using `snapper transform` spell to come close to wolf <-> rat pair and observe.

> items which fall and slide on the ground also slide on water surface
> tried to jump next to a tree (to reach a ledge) but I hit the tree, got stuck in it and then got catapulted 70m into the air (quite funny, though :))

Aware, reproduces, will fix it.

> musik is okay but it seems like the mixing is not right

There is a ticket for it: #117

> walking animation when using keyboard controls but more so when using mouse to look around is "clunky" and stuttery

Aware - that is a big thing, will start a ticket soon

> enemy AI seems to be a little off: combat feels too easy since AI is not fighting back properly

Not reproduced. Maybe combar is easy, because you at start location - monster there are easy to kill

> empty enemy corpses still get highlighted which makes looting enemies on top of each other painful
> when walking against small obstacle like branches and stuff, character bounces off sometimes, which makes him automatically turn around which is quite annoying

Same as in original game, won't fix for now

**Reply by x4rd4s** [reply]
Thank you very much for your reply!

> > no enemy infighting
> 
> Tested today - infighting actually works. I've been using `snapper transform` spell to come close to wolf <-> rat pair and observe.

Yep, you are right. Tested it again and this time I could notice it as well :) 

> > musik is okay but it seems like the mixing is not right
> 
> There is a ticket for it: #117

Ah, yeah. Missed it, thanks!
 
> > enemy AI seems to be a little off: combat feels too easy since AI is not fighting back properly
> 
> Not reproduced. Maybe combar is easy, because you at start location - monster there are easy to kill

I think I just don't suck at combat any more since my last playthrough of Gothic 2 NotR not too far ago.

> > empty enemy corpses still get highlighted which makes looting enemies on top of each other painful
> > when walking against small obstacle like branches and stuff, character bounces off sometimes, which makes him automatically turn around which is quite annoying
> 
> Same as in original game, won't fix for now

I understand but maybe this could be something for the future when all the necessary core stuff is implemented and fixed? I think the corpsehighlighting got adressed in the systempack patch which adds a lot of convenient things to the game and addresses a lot of issues.

Thank you again for your effort. I will test this game further and come back when I have enough stuff on the list :)
